float64 startTime
---
bool success
